docs/evm_training/evmTask4.md

```diff
@@ -1,106 +1,179 @@
+// ./index.js
+const ethers = require('ethers');
 
-### 3. Initiate a Force Bridge Transfer
+async function main() {
+    // Connect to the local network
+    let provider = new ethers.providers.JsonRpcProvider('http://localhost:8545');
 
-Open the [Force Bridge Testnet](https://testnet.forcebridge.com/bridge/Ethereum/Nervos?xchain-asset=0x0000000000000000000000000000000000000000) in a web browser. Make sure you have the Rinkeby network selected in MetaMask and your selected account has Rinkeby ETH.
+    // The address that will receive the contract
+    const receiverAddress = 'YOUR RECEIVER ADDRESS';
 
-You will be transferring from Ethereum to Nervos, and you will be transferring ETH to ckETH. Make sure these values are selected as seen in the screenshots below, if they are not already selected.
+    // ABI of the contract
+    const abi = [
+        // ABI details omitted for brevity
+    ];
 
-<img src={useBaseUrl("img/evmTask4-2.png")}  width="40%"/>
-<img src={useBaseUrl("img/evmTask4-3.png")}  width="40%"/>
+    // Address of the deployed contract
+    const contractAddress = 'YOUR CONTRACT ADDRESS';
 
-After you have selected the assets to transfer across the bridge, you can specify the amount in the top box. There will be a small transfer fee, which will be calculated automatically.
+    // Instantiate a new ethers contract
+    const contract = new ethers.Contract(contractAddress, abi, provider.getSigner());
 
-In the box marked "Recipient", you will specify the Nervos destination address for the funds. Input the deposit receiver address that was generated in the previous step.
+    // Call the contract functions
+    console.log(contract)
+    const balance = await contract.balanceOf(receiverAddress);
+    const totalSupply = await contract.totalSupply();
 
-When you have finished inputting and reviewing your selections, click the `Bridge` button. You will be asked to sign the transaction using MetaMask as seen below.
+    // Log the results
+    console.log('Balance: ', balance.toString());
+    console.log('Total Supply: ', totalSupply.toString());
+}
 
-<img src={useBaseUrl("img/evmTask4-4.png")}  width="40%"/>
+main()
+    .then(() => process.exit(0))
+    .catch(error => {
+        console.error(error);
+        process.exit(1);
+    });
+```
+Save the file and run it using Node.js
+```bash
+node index.js
 
-When you confirm the signing of the transaction in the MetaMask, you will get a modal with a link to the Etherscan explorer, where you can optionally view the progress of mining your transaction on the Ethereum chain.
+Balance:  1000000
+Total Supply:  1000000
+```
 
-<img src={useBaseUrl("img/evmTask4-5.png")}  width="40%"/>
 
-You will also see the transaction at the bottom of the page in the **History** section of the UI under the **Pending** tab. It will first be in a "lock asset on Ethereum (x/12)" state, where X is the number of block confirmations on the Ethereum chain.
+## Deloy Smart Contract on the GodWoken
```

Comment:
```suggestion
## Deploy Smart Contract on the Godwoken testnet
```

Reply:
# Spell Checking

There are many spell checking extensions for VS Code.

The most popular spell checker for VS Code is [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker).

Another one of our favorites is [LTeX](https://marketplace.visualstudio.com/items?itemName=valentjn.vscode-ltex&ssr=false#overview), which is a bit heavier but offers some extra functionality.

Maybe you will be interested in them. @lee920217

Reply:
OK, I will this extension for my editor~